RULES.md

```diff
@@ -275,3 +275,27 @@ Description: Checks that that the ingress permissions on an EC2 Security Group a
 
 Trigger Type: ```Change Triggered```
 Required Parameter: ```None```
+
+### 32. Check that no EC2 is publicly accessible except 80 and 443.
```

Comment:
Please consider updating the descriptions to be "EC2 Instances" where applicable, instead of just "EC2"